### ISSUE TITLE
4.2.0 Compile warnings

### ISSUE THREAD

**Issue by chebur** [adjust/ios_sdk#96]
Having deployment target iOS 7.0+;
```
...Adjust/Adjust/ADJUtil.m:50:10: warning: comparison of address of 'NSURLIsExcludedFromBackupKey' equal to a null pointer is always false [-Wtautological-pointer-compare]
    if (&NSURLIsExcludedFromBackupKey == nil) { // iOS 5.0.1 and lower

...Adjust/Adjust/ADJPackageBuilder.m:222:15: warning: comparison of constant 0 with expression of type 'BOOL' (aka 'bool') is always false [-Wtautological-constant-out-of-range-compare]
    if (value < 0) return;
        ~~~~~ ^ ~
```

**Comment by wellle** [comment]
Thanks, we'll check that.

**Comment by uerceg** [comment]
Closed with PR #98
